tests/tests_suite_test.go

```diff
@@ -304,6 +304,19 @@ func expectRebootedToActive(vm VM) {
 	})
 }
 
+func expectSecureBootEnabled(vm VM) {
+	// Check for secureboot before install, based on firmware env var
+	// if we set, then the test suite will load the secureboot firmware
+	secureboot := os.Getenv("FIRMWARE")
+
+	if secureboot == "true" {
+		By("checking that secureboot is enabled", func() {
+			out, _ := vm.Sudo("dmesge | grep -i secure")
```

Comment:
`dmesge` -> `dmesg` ? This shouldn't work...

Reply:
Manually testing this, very weird indeed because the test actually passed??

Reply:
Ah crap, maybe the if it's not correct so it doesn't go over this...

Reply:
Yes the check is wrong, I'm stupid

Reply:
fixed, good catch @jimmykarily !!